### PR TITLE
Add tax field for businesses

### PR DESCRIPTION
**Pull request by Ray2126**

### CODE REVIEW

File: src/api/accounts/businesses.md [hunk 1]
```diff
@@ -23,20 +23,29 @@ A Business is associated with a single [Account][].
...
+| updatedBy     | {% dt CRN %}          | The User or API Key that updated the Business.      |
+| tax           | [Tax](#tax) {% opt %} | The value-added tax configuration for the Business. |
+
+### Tax
+
+|  Field  |  Type  |                              Description                              |
+| :------ | :----- | :-------------------------------------------------------------------- |
+| value   | String | Value of the tax.                                                     |
+| type    | String | Type of value-added tax. Can be `nz_gst`.                             |
+| country | String | Two-letter ISO code representing the country of the tax. Can be `NZ`. |
```
Comment (Ray2126) [inline src/api/accounts/businesses.md:48]: I know we didn't originally plan to add `type` and `country`, but I don't see the downside of adding them here now. It saves us having to default these fields once we add other options in the future.

Comment (Ray2126) [reply src/api/accounts/businesses.md]: We are moving ahead with `type`, but removing `country`. The `type` indicates what the `country` is anyways.

File: src/api/accounts/businesses.md [hunk 2]
```diff
@@ -23,20 +23,28 @@ A Business is associated with a single [Account][].
...
+| test          | Boolean                             | A flag which is present if the [Account][] is test  |
+| nzbn          | String                              | The unique NZBN identifier.                         |
+| name          | String                              | Legal name recorded in the Companies Register.      |
+| tradingName   | String                              | Trading name recorded in the Companies Register.    |
+| companyNumber | String                              | Company number recorded in the Companies Register.  |
+| createdAt     | {% dt Timestamp %}                  | When the Business was created.                      |
+| updatedAt     | {% dt Timestamp %}                  | When the Business was updated.                      |
+| createdBy     | {% dt CRN %}                        | The User or API Key that created the Business.      |
+| updatedBy     | {% dt CRN %}                        | The User or API Key that updated the Business.      |
+| taxConfig     | [Tax Config](#tax-config) {% opt %} | The value-added tax configuration for the Business. |
```
Comment (tom-blake) [inline src/api/accounts/businesses.md:40]: What does `value-added` mean? Can we just remove it?

Comment (Ray2126) [reply src/api/accounts/businesses.md:40]: It is the specific type of tax. https://en.wikipedia.org/wiki/Value-added_tax

**Review by tom-blake — APPROVED** [review]

**Pull request merged**